packages/EasyStandard/src/Sniffs/Arrays/OrderArrayKeysAlphabeticallySniff.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\Arrays;
+
+use Error;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PhpParser\Comment;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+final class OrderArrayKeysAlphabeticallySniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    public const ARRAY_KEYS_SORT_ALPHABETICALLY = 'ArrayKeysSortAlphabetically';
+
+    /**
+     * @var string
+     */
+    private const COMMENT_CONTENT = '[comment]';
+
+    /**
+     * @var mixed[]
+     */
+    public $skipPatterns = [];
+
+    /**
+     * @param int $bracketOpenerPointer
+     */
+    public function process(File $phpcsFile, $bracketOpenerPointer): void
+    {
+        if ($this->shouldSkip($phpcsFile, $bracketOpenerPointer)) {
+            return;
+        }
+
+        $tokens = $phpcsFile->getTokens();
+        $token = $tokens[$bracketOpenerPointer];
+        $bracketCloserPointer = $token['bracket_closer'] ?? $token['parenthesis_closer'];
+        $code = $phpcsFile->getTokensAsString($bracketOpenerPointer, $bracketCloserPointer - $bracketOpenerPointer + 1);
+
+        $parser = (new ParserFactory())->create(ParserFactory::PREFER_PHP7);
+        try {
+            $ast = $parser->parse('<?php' . PHP_EOL . $code . ';');
+        } catch (Error $error) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                "Parse error: {$error->getMessage()}",
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
+            );
+            $phpcsFile->fixer->endChangeset();
+
+            return;
+        }
+
+        if ($ast === null) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                'Unknown error during parse code',
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
+            );
+            $phpcsFile->fixer->endChangeset();
+
+            return;
+        }
+
+        /** @var \PhpParser\Node\Stmt\Expression $stmtExpr */
+        $stmtExpr = $ast[0];
+        /** @var \PhpParser\Node\Expr\Array_ $array */
+        $array = $stmtExpr->expr;
+
+        $array = $this->refactor($array);
+
+        if ($array === null) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->addErrorOnLine(
+                'Unknown error during refactor array',
+                $token['line'],
+                self::ARRAY_KEYS_SORT_ALPHABETICALLY
```

Comment:
I think we need a separate const for this: `ARRAY_PROCESS_ERROR = 'ArrayProcessError'`.

Reply:
Done.